modules/todo.js

```diff
@@ -80,11 +81,29 @@ export default class Todo {
     exactInput.addEventListener('keyup', this.editTedxt.bind(e, id));
   }
 
+  // checkStatus = (e) => {
+  //   const checkState = e.target.closest('input[type="checkbox"]');
+  //   if (!checkState) return;
+  //   const listLine = checkState.closest('li');
+  //   const { id } = listLine.dataset;
+  //   const status = !!checkState.checked;
+  //   const currentIndex = this.todos.findIndex((todo) => parseInt(todo.id) === parseInt(id));
+  //   listLine.dataset.status = status;
+
+  //   this.todos[currentIndex].status = status;
+  //   localStorage.setItem('todos', JSON.stringify(this.todos));
+  // }
```

Comment:
- [ ] [OPTIONAL] I highly recommend removing commented code from your code base to make your repository look clean and professional. 😄